daisy/step_create_images.go

```diff
@@ -16,63 +16,131 @@ package daisy
 
 import (
 	"context"
+	"encoding/json"
 	"sync"
 
 	"google.golang.org/api/googleapi"
 )
 
+//UseBetaAPI determines if Beta Compute API should be used for calls that use Beta features.
+var UseBetaAPI = false
+
 // CreateImages is a Daisy CreateImages workflow step.
-type CreateImages []*Image
+type CreateImages struct {
+	Images     []*Image
```

Comment:
Due to this change, do we need to change our import/export workflows accordingly?

Reply:
No, because custom Unmarshaller will still parse workflows in original JSON format.